compliance/collection/compliance/node_scanner.go

```diff
@@ -36,10 +35,6 @@ func (n *NodeInventoryComponentScanner) Connect(address string) {
 	if !env.NodeInventoryContainerEnabled.BooleanSetting() {
 		log.Infof("Compliance will not call the node-inventory container, because this is not Openshift 4 cluster")
 	} else if env.RHCOSNodeScanning.BooleanSetting() {
-		// Start the prometheus metrics server
-		metrics.NewServer(metrics.NodeInventorySubsystem, metrics.NewTLSConfigurerFromEnv()).RunForever()
-		metrics.GatherThrottleMetricsForever(metrics.NodeInventorySubsystem.String())
-
```

Comment:
@stehessel This was a duplicate Prom server that was causing the issues. I thought this code was in Scanner container 🤦